packages/studio-ui/src/utils/PageDataValidator.ts

```diff
@@ -8,12 +10,22 @@ export interface ValidationResult {
  * for validation of user-inputted page data.
  */
 export default class PageDataValidator {
-  private isEntityPage = false;
+  private isEntityPage: boolean;
+  private isPagesJSRepo: boolean;
+  private pages: PagesRecord;
 
-  constructor(isEntityPage?: boolean) {
-    if (isEntityPage) {
-      this.isEntityPage = isEntityPage;
-    }
+  constructor({
+    isEntityPage,
+    isPagesJSRepo,
+    pages,
+  }: {
+    isEntityPage: boolean;
+    isPagesJSRepo: boolean;
+    pages?: PagesRecord;
+  }) {
+    this.isEntityPage = isEntityPage ?? false;
+    this.isPagesJSRepo = isPagesJSRepo ?? false;
```

Comment:
since `isEntityPage` and `isPagesJSRepo` are required, we don't need this defaulting

Reply:
oops yeah I made them required to make it less confusing and to get rid of the defaulting-- just forgot to remove that